src/bootstrap/src/lib.rs

```diff
@@ -635,11 +635,12 @@ impl Build {
         if self.config.backtrace {
             features.insert("backtrace");
         }
+
         if self.config.profiler_enabled(target) {
             features.insert("profiler");
         }
-        // Generate memcpy, etc.  FIXME: Remove this once compiler-builtins
-        // automatically detects this target.
+
+        // If zkvm target, generate memcpy, etc.
```

Comment:
This was [asked on the PR](https://github.com/rust-lang/rust/pull/117958#discussion_r1894210041) for clarity, but no response has been provided.

Reply:
cc `riscv32im-risc0-zkvm-elf` target maintainers @flaub, @jbruestle and @SchmErik: could you elaborate why this is needed or if this is still needed?

Note that `riscv32im-risc0-zkvm-elf` is a Tier 3 compiler target, and while we would like to avoid unnecessarily breaking stuff we don't have to, we don't offer any guarantees that it will build. We'd like to trim down complexity in the bootstrap codebase where possible, because it's already very very complex.